fpdcc/ccfp-asset-dashboard
# Add notes field to project model + associated changes

## Overview

Adds a `notes` field to the `Project` model, `ProjectForm`, and CIP export. I adjusted the project edit form layout to make space for the new form field. 

### Demo

<img width="1428" alt="Screenshot 2023-06-05 at 9 49 32 AM" src="https://github.com/fpdcc/ccfp-asset-dashboard/assets/38969506/d1f6a8bd-0905-41df-83ca-044545761eb8">

## Testing Instructions
- edit a project and add some notes text: https://ccfp-asset-d-feature-pr-jsjosl.herokuapp.com/projects/2/
  - login: username=admin && password=password
- visit the CIP export and add your edited project to the cip
- export the CIP selection and make sure the notes column shows up in the export
